pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -743,6 +763,7 @@ && containsRule(ruleSetReferenceId, otherRuleSetReferenceId.getRuleName())) {
      * @return {@code true} if the ruleName exists
      */
     private boolean containsRule(RuleSetReferenceId ruleSetReferenceId, String ruleName) {
+        // unbelievable...
```

Comment:
not helpful...

```suggestion
        // TODO: avoid reloading the ruleset once again
```

I guess, that's what you mean... right?

Reply:
Yeah sorry, I forgot about this... Will create an issue to track this later today